cli/cli/Services/UnityCliGenerator.cs

```diff
@@ -69,7 +69,7 @@ public List<GeneratedFileDescriptor> Generate(CliGeneratorContext context)
 		return files;
 	}
 
-	public static List<Type> RecurseTypes(IEnumerable<Type> inputTypes)
+	public static List<Type> RecurseTypes(IEnumerable<Type> inputTypes, bool includeTypesFromInvalidAssembly=false)

```

Comment:
Where is this method being called with the new `includeTypesFromInvalidAssembly ` parameter? Couldn't find it, is this being called by reflection or something like that?

Reply:
I saw it in the test, but in this question I mean in the actual generation for Unity.

Reply:
see `UnityCliGenerator` line 44

Reply:
I see, but that is not sending the new parameter, right? So we are adding this just as an option, that we are not using right now, right?